Rover/GCS_MAVLink_Rover.cpp

```diff
@@ -578,6 +585,38 @@ MAV_RESULT GCS_MAVLINK_Rover::handle_command_nav_set_yaw_speed(const mavlink_com
 }
 #endif
 
+MAV_RESULT GCS_MAVLINK_Rover::handle_command_condition_yaw(const mavlink_command_int_t &packet)
+{
+        // param1 : target angle [0-360]
+        // param2 : speed during change [deg per second]
+        // param3 : direction (-1:ccw, +1:cw, 0:quickest direction to target heading)
+        // param4 : relative offset (1) or absolute angle (0)
+
+        // exit if vehicle is not in Guided mode
+        if (!rover.control_mode->in_guided_mode()) {
+            return MAV_RESULT_FAILED;
+        }
+
+        // Check valid direction parameter
+        int16_t direction = static_cast<int16_t>(packet.param3);
+        if (direction != 0 && direction != 1 && direction != -1) {
```

Comment:
```suggestion
        if (abs(direction) > 1) {  // only 0, -1 and 1 are valid direction values
```

Very much optional change.

Reply:
Way more elegant :) . Done